src/server/engine_shard_set.h

```diff
@@ -257,8 +251,8 @@ void EngineShardSet::RunBriefInParallel(U&& func, P&& pred) const {
 
     bc.Add(1);
     util::ProactorBase* dest = pp_->at(i);
-    dest->DispatchBrief([f = std::forward<U>(func), bc]() mutable {
-      f(EngineShard::tlocal());
+    dest->DispatchBrief([&func, bc]() mutable {
+      func(EngineShard::tlocal());
```

Comment:
I'm not sure, should we capture by value or just take the reference? I've skimmed through the usages and didn't find anything that requires strict copying

Reply:
If someone creates a lambda with a mutable member that is re-assigned inside the lambda he'll have a bad surprise. Otherwise there is no reason for defensive copies